template-helpers/emails.js

```diff
@@ -1,25 +1,165 @@
+const modifiedStrings = {
+  "pwt-summary-2" : "fxa-pwt-summary-2",
+  "pwt-summary-4" : "fxa-pwt-summary-4",
+  "pwt-summary-6" : "fxa-pwt-summary-6",
+  "what-to-do-subhead-2" : "fxa-what-to-do-subhead-2",
+  "what-to-do-subhead-4" : "fxa-what-to-do-subhead-4",
+  "what-to-do-blurb-1" : "fxa-what-to-do-blurb-1",
+  "what-to-do-blurb-2" : "fxa-wtd-blurb-2",
+  "what-to-do-blurb-3" : "fxa-what-to-do-blurb-3",
+  "what-to-do-blurb-4" : "fxa-what-to-do-blurb-4",
 };
 
+
+function getUnsafeBreachesForEmailReport(args) {
+  const locales = args.data.root.supportedLocales;
+  const foundBreaches = args.data.root.unsafeBreachesForEmail;
+
+  const breachCards = makeBreachCards(foundBreaches, locales);
+  return breachCards;
+}
+
+
+function makeFaqLink(target, campaign) {
+  const url = new URL(`https://support.mozilla.org/kb/firefox-monitor-faq${target}`);
+  const utmParameters = {
+    utm_source: "fx-monitor-email",
+    utm_medium: "email",
+    utm_campaign: campaign,
+  };
+
+  for (const param in utmParameters) {
+    url.searchParams.append(param, utmParameters[param]);
+  }
+  return url;
+}
+
+
+function getBreachAlertFaqs(args) {
+  const supportedLocales = args.data.root.supportedLocales;
+  const faqs = [
+    {
+      "stringId" : "faq1",
+      "stringDescription": "I don't recognize this website. Why am I in this breach?",
+      "href": makeFaqLink("#w_i-donaot-recognize-this-company-or-website-why-am-i-receiving-notifications-about-this-breach", "faq1"),
+    },
+    {
+      "stringId" : "faq2",
+      "stringDescription": "Why did it take so long to notify me of this breach?",
+      "href": makeFaqLink("#w_why-did-it-take-so-long-to-notify-me-of-this-breach", "faq2"),
+    },
+    {
+      "stringId" : "faq3",
+      "stringDescription": "How do I know this is a legitimate email from Firefox Monitor?",
+      "href": makeFaqLink("#w_how-do-i-know-these-emails-are-really-from-firefox-and-not-from-a-hacker", "faq3"),
+    },
+  ];
+  faqs.forEach(faq => {
+    faq.linkTitle = LocaleUtils.fluentFormat(supportedLocales, faq.stringId, args);
+  });
+
+  let result = "";
+  for (let i = 0; i < faqs.length; i++) {
+    result = result + args.fn(faqs[i]);
+  }
+  return result;
+}
+
+function getReportHeader(args) {
+  const locales = args.data.root.supportedLocales;
+  const reportHeader = {
+    date: prettyDate(new Date(), locales),
+    strings: {
+      "email": "email-address",
+      "reportDate": "report-date",
+      "fxmReport" : "firefox-monitor-report",
+    },
+  };
+  for (const stringId in reportHeader.strings) {
+    const newString = LocaleUtils.fluentFormat(locales, reportHeader.strings[stringId]);
+    reportHeader.strings[stringId] = newString;
+  }
+  return args.fn(reportHeader);
+}
+
+
+function getWhatToDos(args) {
+  const locales = args.data.root.supportedLocales;
+
+  let headlineId = "what-to-do-next";
+  let subheadId = "what-to-do-subhead-";
+  let blurbId = "what-to-do-blurb-";
+  let numTips = 4;
```

Comment:
Lots of `let` assignments here "smells" like it could be improved, but I'm not sure how. No biggies unless you can think of a quick way to do it.